src/guessWords/containers/Input.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, Fragment, useCallback } from "react";
+
+function Input(props) {
+  const [guessedWord, setGuessedWord] = useState("");
+  const onChangeCallback = useCallback((event) => {
+    setGuessedWord(event.target.value);
+  });
```

Comment:
add an empty array to dependencies of useCallback
```suggestion
  }, []);
```